lib/circuitbox/faraday_middleware.rb

```diff
@@ -31,7 +31,7 @@ def initialize(response = nil, exception = nil)
     }.freeze
 
     DEFAULT_EXCEPTIONS = [
-      Faraday::Error::TimeoutError,
+      Faraday::TimeoutError,
```

Comment:
This means it will no longer work with pre `1.0` faraday, I guess there is no way around it but I think there should be a note in the readme somewhere to make sure users are aware, as faraday by default is only a development dependency as one should bring their own faraday in case of using the middleware.

Alternatively we could check if `Faraday::TimeoutError`  is defined and add the appropriate one?

Reply:
The alternative is pivoting on `Faraday::VERSION` but I agree something like this would be good:

```ruby
defined?(Faraday::TimeoutError) ? Faraday::TimeoutError : Faraday::Error::TimeoutError
```

Definitely should go in the changelog. Most of the Faraday 1.x upgrades have been really painful for me so far.